packages/core/src/App/Containers/Layout/header/header.tsx

```diff
@@ -84,11 +89,13 @@ const Header = observer(() => {
             case pathname === routes.onboarding:
                 result = null;
                 break;
+            case is_dtrader_v2_enabled && is_mobile && pathname.startsWith(routes.trade):
+                result = <DTraderV2Header />;
+                break;
             case is_dtrader_v2_enabled &&
                 is_mobile &&
-                (pathname.startsWith(routes.trade) ||
-                    pathname.startsWith('/contract/') === routes.contract.startsWith('/contract/')):
-                result = <DTraderV2Header />;
+                pathname.startsWith('/contract/') === routes.contract.startsWith('/contract/'):
```

Comment:
When from closed positions I'm going to Contract details and then back, should I arrive to closed positions or to open positions? Because on the test link I'm going to open positions. Is it expected?

Reply:
We call history.goBack on the back button . The route for closed/open position is same and it redirects to that route . I am not sure if we have to change tabs when we go to click back in case where user came from closed positions.

Reply:
Sure, I was just asking) I know, that open/close tabs are not about routing, but we can keep somewhere in our new fancy story flag for recently opened tab. Anyway, we can add it later if it will be among the requirements. Thank you for checking)